### DISCUSSION TITLE
what's the difference on CRDs between Erie Canal and MCS API?

### DISCUSSION THREAD

**Discussion by addozhang** [flomesh-io/ErieCanal#26] [Q&A] [answered]
**Please describe what should be documented**
It's described that Erie Canal is an implementation of MCS API, but it uses CRDs from a different group. 

Could you figure out their difference and where it comes from?
**Please suggest where in the repo the document should be located**

**Comment by i0r3k** [comment, marked as answer]
#  ServiceExport
<img width="508" alt="image" src="https://user-images.githubusercontent.com/78458661/210201803-0fcb7082-1da4-4688-bbf4-253311704a23.png">

# ServiceImport
<img width="984" alt="image" src="https://user-images.githubusercontent.com/78458661/210201868-1400cd59-1563-417e-a5d5-bf24e298f44c.png">

**Comment by addozhang** [comment]
Thanks @reaver-flomesh .
